ansible/workers.yml

```diff
@@ -0,0 +1,116 @@
+---
+- hosts:
+  - worker_rhel6
+  - worker_rhel7
+  - worker_fedora
+
+  become: yes
+  become_method: sudo
+
+  tasks:
+  - name: Set up Jenkins user
+    user:
+      name: jenkins
+      comment: "Jenkins user"
+
+  # we use the public key to connect to slaves from jenkins master
+  - name: Authorize master's jenkins user public key
+    authorized_key:
+      user: jenkins
+      key: "{{ lookup('file', 'generated_bits/master_id_rsa.pub') }}"
+      exclusive: yes  # only need one key for "jenkins" user, users can still connect as ec2-user
+      state: present
+
+  - name: Ensure general Jenkins dependencies are installed
+    package:
+      name:
+        - java
+
+  - name: Ensure general build dependencies are installed
+    package:
+      name:
+        - autoconf
+        - automake
+        - libtool
+        - doxygen
+        - git
+        - rubygems
+        - gawk
+
+  - name: Ensure OpenSCAP dependencies are installed
+    package:
+      name:
+      - dbus-devel
+      - libacl-devel
+      - libblkid-devel
+      - libcap-devel
+      - libcurl-devel
+      - libgcrypt-devel
+      - libselinux-devel
+      - libxml2-devel
+      - libxslt-devel
+      - make
+      - openldap-devel
+      - pcre-devel
+      - perl-XML-Parser
+      - perl-XML-XPath
+      - perl-devel
+      - python-devel
+      - rpm-devel
+      - swig
+      - bzip2-devel
+      - GConf2-devel
+      - sendmail  # the openscap OVAL mitre requires sendmail to complete
```

Comment:
@matusmarhefka yeah that looks like the way to do it, I would also comment in the name that this is an OpenSCAP dep required on the workers.